ibis-server/app/model/metadata/trino.py

```diff
@@ -38,7 +38,7 @@ def get_table_list(self) -> list[Table]:
                     AND t.table_name = c.table_name
                 INNER JOIN
                     system.metadata.table_comments AS tc
-                    ON t.table_catalog = c.table_catalog
+                    ON t.table_catalog = tc.catalog_name
```

Comment:
I found this change isn't covered by the current tests. Even if I roll back the change, the Trino test is pass. I think it would be better if we provide a new test covering this change.

Reply:
Thanks for the heads up! I didn't realize this bug was only in the `tpch.tiny.customer` table. I'll update the test cases to reflect that.